Tests/SwiftProtobufTests/Test_AllTypes_Proto3.swift

```diff
@@ -777,13 +777,17 @@ class Test_AllTypes_Proto3: XCTestCase, PBTestHelpers {
         assertDecodeFails([253, 1, 77])
         assertDecodeFails([254, 1, 78])
         assertDecodeFails([255, 1, 79])
-        assertDebugDescription("Proto3TestAllTypes(repeatedInt32:[1])") {(o: inout MessageTestType) in
+        // WHY ISN'T THIS LIKE PROTO2?
+        //   "Proto3TestAllTypes:\nrepeated_int32: 1\n"
+        assertDebugDescription("Proto3TestAllTypes:\nrepeated_int32: [1]\n") {(o: inout MessageTestType) in
```

Comment:
Two catches:
1. A proto3 syntax files has no control over if things are packed or not.
1. packed just controls how well behaved encoders write out the bytes, they have to be able to read both forms.

So I'm not so sure the end developers would understand the distinction.  But they also aren't likely to be comparing dumps and notice the difference.  😄   Since I was updating tests, it stuck out as I had to make otherwise identical changes in the files.